portal-bridge/src/main.rs

```diff
@@ -56,7 +56,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         let (census_tx, census_rx) = mpsc::unbounded_channel();
         let mut census = Census::new(portal_client.clone(), census_rx);
         // initialize the census to acquire critical threshold view of network before gossiping
-        census.init().await;
+        if census.init().await.is_err() {
```

Comment:
nit: In my opinion, main can just return `anyhow::Result<()>` and you can use `anyhow!(..)` here. Or just use `census.init().await?;` and propagate error.